src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -278,7 +277,9 @@
       [quo/community-tags
        {:tags            tags
         :last-item-style style/last-community-tag
-        :container-style style/community-tag-container}]
+        :container-style (if (> (count tags) 0)
```

Comment:
I like `cond->` to describe optional transformations starting from a known good value. For this particular code it would be (untested):

```clojure
(cond-> style/community-tag-container
  (pos? (count tags))
  (assoc :margin-bottom 20))
```

Reply:
Hi @ilmotta good point, I don't know who is the design team, can you reference me someone, please

Reply:
Hey @andresceballosm, sorry I assumed you already had a contact. The Design Team is the one responsible for all things Figma, and in GitHub, you can reach out to @Francesca-G. Thank you.

Reply:
@andresceballosm, I'll run another review in your PR today after Francesca confirmed the requirements.

Reply:
Seems reasonable @andresceballosm. I would double-check the UI again because I'm never certain if there's a parent/sibling component that might affect *something*, *somewhere* you know.